reV/econ/econ.py

```diff
@@ -141,9 +144,13 @@ def __init__(self, points_control, cf_file, cf_year, site_data=None,
         self._out_chunk = ()
         self._init_out_arrays()
 
-        # initialize output file
-        self._init_fpath()
-        self._init_h5()
+        # initialize output file or append econ data to gen file
+        if append:
+            self._fpath = self._cf_file
+        else:
+            self._init_fpath()
+        mode = 'a' if append else 'w'
+        self._init_h5(mode=mode)
```

Comment:
I don't understand how this doesn't break? Output.init_h5 will try to create all of the datasets which will break if they already exist. Am I missing something?

Reply:
Econ should only be making new datasets, plus I added better handling of pre-existing datasets.